docs/using-az-aro.md

```diff
@@ -66,9 +66,13 @@ subscription, ensure that the `Microsoft.RedHatOpenShift` resource provider is
 registered:
 
 ```
-az provider register -n Microsoft.RedHatOpenShift --wait
+az feature register --subscription <Your Subscription ID> --name preview --namespace Microsoft.RedHatOpenShift
```

Comment:
@julienstroheker please could you explain this change?  I did `az provider register` at the time - what would `az feature register` do?